lib/buffer.js

```diff
@@ -535,10 +535,35 @@ Buffer.prototype.inspect = function inspect() {
 };
 
 
-Buffer.prototype.compare = function compare(b) {
+Buffer.prototype.compare = function compare(b, start, end, thisStart, thisEnd) {
   if (!(b instanceof Buffer))
     throw new TypeError('Argument must be a Buffer');
 
+  if (typeof start !== 'undefined' &&
+      (typeof start !== 'number' ||
+       +start !== (start >> 0)))
```

Comment:
I thought we were adding braces to conditionals that spanned multiple lines themselves, despite the number of lines in their actual blocks?